framework/include/base/Assembly.h

```diff
@@ -959,16 +959,80 @@ class Assembly
    */
   DenseMatrix<Number> & jacobianBlock(unsigned int ivar, unsigned int jvar, TagID tag = 0)
   {
-    _jacobian_block_used[tag][ivar][jvar] = 1;
+    jacobianBlockUsed(tag, ivar, jvar, true);
     return _sub_Kee[tag][ivar][_block_diagonal_matrix ? 0 : jvar];
   }
 
+  /**
+   * Flags to check whether or not Jacobian blocks are used
```

Comment:
I would change this comment to say: `Sets whether or not Jacobian is used to \p used`

Reply:
Done